sympy/solvers/ode/tests/test_systems.py

```diff
@@ -698,10 +706,114 @@ def test_sysode_linear_neq_order1():
              Eq(h(t), C2)]
 
     assert dsolve(eq25) == sol25
-    assert checksysodesol(eq25, sol25)
+    assert checksysodesol(eq25, sol25) == (True, [0, 0, 0, 0, 0])
+
+    eq26 = [Eq(diff(f(t), t), 2*f(t)), Eq(diff(g(t), t), 3*f(t) + 7*g(t))]
+    sol26 = [Eq(f(t), -5*C1*exp(2*t)/3), Eq(g(t), C1*exp(2*t) + C2*exp(7*t))]
+    assert dsolve(eq26) == sol26
+    assert checksysodesol(eq26, sol26) == (True, [0, 0])
+
+    eq27 = [Eq(diff(f(t), t), -9*I*f(t) - 4*g(t)), Eq(diff(g(t), t), -4*I*g(t))]
+    sol27 = [Eq(f(t), C1*exp(-9*I*t) + 4*I*C2*exp(-4*I*t)/5), Eq(g(t), C2*exp(-4*I*t))]
+    assert dsolve(eq27) == sol27
+    assert checksysodesol(eq27, sol27) == (True, [0, 0])
+
+    eq28 = [Eq(diff(f(t), t), -9*I*f(t)), Eq(diff(g(t), t), -4*I*g(t))]
+    sol28 = [Eq(f(t), C1*exp(-9*I*t)), Eq(g(t), C2*exp(-4*I*t))]
+    assert dsolve(eq28) == sol28
+    assert checksysodesol(eq28, sol28) == (True, [0, 0])
+
+    corner_cases = [(0, 0, 0, 0), (1, 0, 0, 0), (0, 1, 0, 0),
+                    (0, 0, 1, 0), (0, 0, 0, 1), (1, 0, 0, I),
+                    (I, 0, 0, -I), (0, I, 0, 0), (0, I, I, 0)]
```

Comment:
I think it would be better to spell these examples out explicitly (like e.g. eq28). There are good ways to reduce repetition in the ODE tests like in `test_single.py`. I think it's important for the equations and solutions to spelled out explicitly and clearly though for someone reading the tests.

Reply:
Ok, I would write them out as individual test cases.